benchopt/cli.py

```diff
@@ -19,6 +19,9 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 
+PLOT_KINDS_NAMES = list(PLOT_KINDS.keys())
```

Comment:
To make sure the names are all formatted the same (even the last one which is not processed at the same time), you could do:
```suggestion
PLOT_KINDS_NAMES = list(f'`{k}`' for k in PLOT_KINDS)
```
and then use the name directly in `join` and for the last one.